src/app/containers/LoginPage/index.js

```diff
@@ -61,6 +98,26 @@ function LoginPage({ history }) {
     [data],
   );
 
+  const handleChange = (name, value) => {
+    setData({ ...data, [name]: value });
+    setErrors({ ...errors, [name]: null });
+  };
+
+  const defaultProps = (name, value) => ({
+    label: labels[name],
+    name,
+    value,
+    onChange: (name, value) => handleChange(name, value),
+    error: errors[name],
+  });
+
+  const selectContent = (showPopup) => {
+    console.log(showPopup);
+    if (showPopup === POPUP.error) return POPUP_MSG.error;
+    if (showPopup === POPUP.warning) return POPUP_MSG.warning[0];
+    return null;
```

Comment:
there are more than 2 errors, cannot use `showPopup` to determine the error message